server/middleware/checkPrisonerInCaseloadMiddleware.test.ts

```diff
@@ -72,7 +72,7 @@ describe('checkPrisonerInCaseloadMiddleware', () => {
     // Given
     const middleware = checkPrisonerInCaseload({ allowGlobal: true })
 
-    const user = hmppsUser({ activeCaseLoadId: 'LEI', caseLoadIds: ['LEI'], userRoles: ['ROLE_GLOBAL_SEARCH'] })
+    const user = hmppsUser({ activeCaseLoadId: 'LEI', caseLoadIds: ['LEI'], roles: ['ROLE_GLOBAL_SEARCH'] })
```

Comment:
In my last PR that added the `checkPrisonerInCaseloadMiddleware` stuff, where I'd copied it from DPS, I was using the wrong property name from the user in session re: their roles.
In the DPS codebase they use the property name `userRoles`, but our `User` uses the property name `roles` (`UserDetails` in `userService.ts`)